tv_apps/smart-system/js/crash_reporter.js

```diff
@@ -69,8 +69,10 @@ window.CrashReporter = (function() {
                addEventListener('action', function onAction() {
```

Comment:
I'd prefer putting the "dot" right before `addEventListener` for better readability.